SQL_assignment/sql_assignment1

```diff
@@ -0,0 +1,82 @@
+CREATE TABLE: Creates an Employees table with columns for EmployeeID, FirstName, LastName, Department, and Salary.
+
+    Create table emp(EmpoyeeID int primary key,firstname varchar(20),lastname varchar(20),department varchar(20),salery int)
+
+
+
+INSERT: Adds three employee records to the table.
+
+    insert into emp values(1,'akash','j','IT',50000)
```

Comment:
Whenever we are inserting data into a table, it is always better to include the column list as well.
like below.

insert into emp (empId, firstname, lastname, department, salary) values(1,'akash','j','IT',50000)

Reply:
okey noted